KeenClient/KIODBStore.m

```diff
@@ -127,6 +143,8 @@ - (BOOL)openAndInitDB {
 - (void)closeDB {
     // Free all the prepared statements. This is safe on null pointers.
     if(dbIsOpen) {
+        self.dbQueue = nil;
```

Comment:
So dispatching will fail immediately once this is set, right? Is that handled OK in the rest of this code?